### PR TITLE
Blog screen developement

### PR DESCRIPTION
**Pull request by zmagar**
## Description
Implemented async function to fetch the API data to better handle the data and errors.
Implemented the loading indicator to show while data is being fetched.
Updated images matching blog entries from the updated API
Implemented back arrow button in the innerAppBrowser instead of X
Styling changes (picture size, font size)

## To Test
1. Switch to ab-46-blogscreen-developement.
2. npm install
3. Run the app
4. Go to blog screen, check if loading indicator is there, if links are working, etc.

### CODE REVIEW

File: components/screens/BlogScreen/BlogScreen.js [hunk 1]
```diff
@@ -1,62 +1,98 @@
...
+                />
+                <Text key={body} style={Styles.blogItem} >{body}</Text>
+                <View key={url} style={{ width: 200, height: 50 }}>
+                  <Button
+                    style={Styles.button}
+                    type="solid"
+                    title="Read more"
+                    color="#F55243"
+                    size={6}
+                    onPress={(e) => {
```
Comment (nataliagourova) [inline components/screens/BlogScreen/BlogScreen.js:88]: As a suggestion, type -> color could be moved to the Styles sheet, as button or button2.

**Review by nataliagourova — CHANGES_REQUESTED** [review]
Nice use of Activity Indicator and InAppBrowser async handling.

File: components/screens/BlogScreen/BlogScreen.js [hunk 2]
```diff
@@ -1,62 +1,98 @@
...
+        enableUrlBarHiding: true,
+        enableDefaultShare: true,
+        forceCloseOnRedirection: false,
+        hasBackButton: true,
+      })
+    }
+    else Linking.openURL(urlToOpen)
+  } catch (error) {
+    console.log(error.message)
+  }
```
Comment (CalebMcOlin) [inline components/screens/BlogScreen/BlogScreen.js:29]: Just want to say I love this! Looks very slick and professional.

I think once we get all our screens pushed to main we should implement this or something similar for all our InAppBrowsers!

**Review by CalebMcOlin — DISMISSED** [review]

File: components/screens/BlogScreen/BlogScreen.js (hunk 1 above)
Comment (zmagar) [reply components/screens/BlogScreen/BlogScreen.js]: Thank you Natalia, moved it to the styles sheet!

File: components/screens/BlogScreen/BlogScreen.js (hunk 2 above)
Comment (zmagar) [reply components/screens/BlogScreen/BlogScreen.js:29]: Thank you Caleb!

**Review by nataliagourova — APPROVED** [review]
Approving here for merging with main, but did not see a comment on Alert import (are we using it or not?), please address in a new PR if needed.

**Comment by zmagar** [comment]
Thank you @nataliagourova for pointing out at unused Alert component, I will fix it in my next PR

**Pull request merged**